portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,34 @@ function addFunFacts() {
     const factsContainer = document.getElementById('facts-container');
     factsContainer.innerText = fact;
 }
+
+
+/** Fetches the current date from the server and adds it to the page. */
+// async function showServerTime() {
+//   const responseFromServer = await fetch('/date');
+//   const textFromResponse = await responseFromServer.text();
+
+//   const dateContainer = document.getElementById('date-container');
+//   dateContainer.innerText = textFromResponse;
+// }
+
+/** Fetches stats from the server and adds them to the page. */
+async function getFacts() {
+  const responseFromServer = await fetch('/facts');
+  // The json() function returns an object that contains fields that we can
+  // reference to create HTML.
+  const stats = await responseFromServer.json();
+
+  // Pick a random fact.
+  const quotes = stats[Math.floor(Math.random() * stats.length)];
+
+  const statsListElement = document.getElementById('quote-container');
+  statsListElement.innerHTML = quotes;
+}
+
+/** Creates an <li> element containing text. */
+function createListElement(text) {
```

Comment:
Is this still needed? I don't see any references to it. If you do need it for future work, feel free to leave it here.

Reply:
I will remove it. Just figured out that it isn't needed.